### DISCUSSION TITLE
Card Colapsado e Expandido

### DISCUSSION THREAD

**Discussion by anapego** [TiendaNube/nimbus-design-system#181] [Ideas] [answered]
![card colapsado](https://github.com/TiendaNube/nimbus-design-system/assets/92194136/7ef4560b-7229-4ea5-adcf-232ee99f2fce)
[card colapsado@1 .pdf](https://github.com/TiendaNube/nimbus-design-system/files/12034498/card.colapsado%401.pdf)

**Comment by jcampos89** [comment, marked as answer]
@anapego obrigado por enviar sua sugestão! Conversando com o time chegamos a conclusão de que o cenário mais a direita é viável, não existe no nosso UI kit porém é perfeitamente viável no código. para reproduzir você pode construir um componente local e aplicá-lo dentro do card.
